.github/workflows/main.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: ['3.6', '3.7', '3.8', '3.9', 'pypy-3.6']
+        python-version: ['3.7', '3.8', '3.9', '3.10']
```

Comment:
This was just wrong, but wasn't an issue until the builds started using `3.10` by default.